content/en/tracing/trace_collection/automatic_instrumentation/single-step-apm.md

```diff
@@ -59,7 +59,7 @@ For a Docker Linux container:
 
 1. Run the one-line installation command:
    ```shell
-   bash -c "$(curl -L https://install.datadoghq.com/scripts/install_script_docker_injection.sh)"
+   DD_APM_INSTRUMENTATION_ENABLED=docker bash -c "$(curl -L https://install.datadoghq.com/scripts/install_script_agent7.sh)"
```

Comment:
oh I just noticed that to keep this equivalent to the previous command we also need to pass `DD_INSTALL_ONLY=true` to not install the host agent

Reply:
https://github.com/DataDog/documentation/pull/24401